source/sync/data-model/create-a-schema.txt

```diff
@@ -271,44 +242,15 @@ Create a Realm Object Schema from an App Services Schema
                   // ... use the realm instance to read and modify data
                 })
 
-Refer to the Realm SDK documentation to use the generated Realm Object Schema.
-
-.. tabs-realm-sdks::
-
-   .. tab::
-      :tabid: android
-
-      :ref:`Sync Data with the Realm Java SDK <java-sync-data>`
-
-   .. tab::
-      :tabid: ios
-
-      :ref:`Sync Data with the Realm Swift SDK <ios-sync-changes-between-devices>`
-
-   .. tab::
-      :tabid: node
-
-      :ref:`Sync Data with the Realm Node.js SDK <node-sync-changes-between-devices>`
-
-   .. tab::
-      :tabid: react-native
-
-      :ref:`Sync Data with the React Native SDK <react-native-realm-sync>`
-
-   .. tab::
-      :tabid: dotnet
-
-      :ref:`Sync Data with the Realm .NET SDK <dotnet-sync-changes-between-devices>`
-
-   .. tab::
-      :tabid: flutter
-
-      :ref:`Sync Data with the Realm Flutter SDK <flutter-sync>`
-
-   .. tab::
-      :tabid: kotlin
+Refer to the Realm SDK-specific documentation to use the generated Realm Object Schema.
 
-      :ref:`Sync Data with the Realm Kotlin SDK <kotlin-sync-overview>`
+- :ref:`Realm Java SDK <java-sync-data>`
+- :ref:`Realm Swift SDK <ios-sync-changes-between-devices>`
+- :ref:`Realm Node.js SDK <node-sync-changes-between-devices>`
+- :ref:`Realm React Native SDK <react-native-realm-sync>`
+- :ref:`Realm .NET SDK <dotnet-sync-changes-between-devices>`
+- :ref:`Realm Flutter SDK <flutter-sync>`
+- :ref:`Realm Kotlin SDK <kotlin-sync-overview>`
```

Comment:
Should we alphabetize?
```
- :ref:`Realm .NET SDK <dotnet-sync-changes-between-devices>`
- :ref:`Realm Flutter SDK <flutter-sync>`
- :ref:`Realm Java SDK <java-sync-data>`
- :ref:`Realm Kotlin SDK <kotlin-sync-overview>`
- :ref:`Realm Node.js SDK <node-sync-changes-between-devices>`
- :ref:`Realm React Native SDK <react-native-realm-sync>`
- :ref:`Realm Swift SDK <ios-sync-changes-between-devices>`
```

Reply:
Ah, yeah, that probably makes sense. I'll update the order.